recipes/perl-threaded/meta.yaml

```diff
@@ -1,21 +1,15 @@
-# Derived from the dan_blanchard channel.
+# This is a meta-package installing the threaded version of PERL from conda-forge
+
 package:
   name: perl-threaded
   version: 5.22.0
 
-source:
-  fn: perl-5.22.0.tar.gz
-  url: http://www.cpan.org/src/5.0/perl-5.22.0.tar.gz
-  md5: e32cb6a8dda0084f2a43dac76318d68d
-
 build:
-  number: 10
-  skip: False
+  number: 11
 
 requirements:
-  build:
-    - gcc   # [linux]
-    - llvm  # [osx]
+  run:
+    - perl
```

Comment:
Yes, we can do this. I not able to do it today. @nsoranzo feel free to create a PR